src/main/java/org/jabref/FallbackExceptionHandler.java

```diff
@@ -17,5 +19,6 @@ public static void installExceptionHandler() {
     @Override
     public void uncaughtException(Thread thread, Throwable exception) {
         LOGGER.error("Uncaught exception occurred in " + thread, exception);
+        new FallbackErrorView().show();
```

Comment:
That's a bit more complicated than it need be. With 
https://github.com/JabRef/jabref/blob/a4e735b3870a329c8d5d8de443a576a4b0ad586c/src/main/java/org/jabref/gui/DialogService.java#L71-L77
we already provide the facilities to show an error dialog. You can get the DialogService via `JabRefGUI.getMainFrame().getDialogService()`

Reply:
One question from the off @tobiasdiez : Should really a dialog be displayed in case of an exception caught be the fallback exception handler? Or should we just display a short "bubble information" (sorry I don't know the right term atm ;-)) as for "saving" etc.?

Reply:
In my opinion, every exception that is unhandled and reaches the fallback exception handler is a bug. I guess you normally want to catch exceptions earlier and then decide with the help of context information how to inform the user about it (ignore, warn, issue a user friendly message, ...). Thus, I would show the full exception log and hope that user report these errors when they are reproducible so that we can fix them. What do you think?

Reply:
Yes, I agree that these exceptions are bugs. 
And therefore I'm not sure whether we should really bother the user with a dialog he needs to click away - or whether the information "Oh there seems to be something wrong, if you want to see more detail go to the event log" that just pops up for a few seconds is less annoying.

Seems to depend on what actually is broken: In case normal working is still possible and the triggered functionality is still *somehow* working the bubble dialog would be sufficient and less annoying - however, if the functionality is completely broken (and for example the event log window cannot be opened anymore) the standard dialog is clearly better.

.... okay, the more I write and think about here, the more I come to the conclusion that the approach with a "real" dialog is the better approach in order to get better bug reports here at GitHub.

Reply:
Now the exception and the message are shown using showErrorDialogAndWait.